_pages/learn.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: splash
+permalink: /learn.html
+title: "Python Tutorials & Learning Resources for Scientists"
+classes: flowing
+header:
+  overlay_image: images/header.png
+  overlay_filter: 0.6
+scientists:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Our catalog of vetted open source tools makes it easier for scientists to find the trusted tools that they need to develop their open science workflows."
+    title: "Help scientists find the open science tools that they need"
+    url: https://www.pyopensci.org/python-packages.html
+    btn_label: View our growing list of accepted scientific Python packages
+    btn_class: btn--primary
+overview:
+  - excerpt: "pyOpenSci creates resources to help you navigate the Python packaging ecosystem with ease. All of our materials are community developed and go through extensive review."
+development:
+   - image_path: images/pyopensci-learn-community-created.png
+     alt: "alt here "
+     title: "Community Developed Python Tutorials for Scientists"
+     excerpt: "Our tutorials are created through a multi-stage community review process.
+
+     * Tutorials are developed by the pyOpenSci team or community members.
+
+     * Tutorials are reviewed by tool maintainers to ensure ideas and concepts are accurate
+
+     * Before publication, tutorials then go through several rounds of community review for accuracy, usability and accessibility."
+whats_next:
+  - image_path: images/pyopensci-learn-github-collaboration.png
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Through the Better software for science fellowship we will be developing lessons focused on teaching core Github collaboration skills needed for both contributing to open source software and for working in an open science team environment. "
+    title: "What's next: Collaborative GitHub for Scientists"
+toc: false
+---
+
+{% include feature_row id="overview" type="center" %}
+
+<div class="pyos-section purple">
+<div class="content" markdown="1">
+
+{% include feature_row id="development" type="left" %}
+
+</div>
+</div>
+
+{% include div_purple_bottom.html  %}
+
+<div class="pyos-section">
+<div class="content" markdown="1">
+
+## Beginner friendly Python packaging tutorials
+
+Beginner-friendly Python packaging tutorials that will take you through
```

Comment:
it's odd that this text randomly breaks mid-page, rather than extending the full width of the header

Reply:
yep, that's what I'm referring to! if our page is responsive, then it shouldn't matter how many characters there are to a line for a subheader, because people can (and will) resize their screens to whatever is comfortable for them.

if it were a longer piece of text we'd maybe want to consider implementing something, but I don't think it's necessary here.

Reply:
ok so look at this @kierisi 
<img width="1278" alt="Screenshot 2024-04-02 at 10 51 51 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/a2e0d863-ecf2-440d-a737-00b09943189c">

1. i did try to make the two sections more clearly different - let me know if that is better BUT ALSO
2. notice how the left and right alignment is DIFFERENT FOR EACH

i'll play with this a bit

Reply:
I wonder if it's a div, or however you've got the purple background set up.

Reply:
Now i'm noticing the h2 tags are different sizes 😿 this is because i've hacked this site together over minimal mistakes so that is what is happening here. 
<img width="1240" alt="Screenshot 2024-04-02 at 11 13 47 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/f893a5c2-c739-406f-8795-2d9bbdf24d49">

Reply:
it is. the top div is using the minimal mistakes styles - those feature-wrapper, feature-content etc. 
<img width="693" alt="Screenshot 2024-04-02 at 11 21 49 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/b3687212-6237-4f85-b7ac-7956763122ec">

And the bottom one are div styles i created for pyOS. the top is a templated include. i am not sure how to do that with a liquid section that creates cards via it's own include. i don't know if you can (or want to) pass an include with liquid to an include if that makes sense. 

<img width="832" alt="Screenshot 2024-04-02 at 11 21 57 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/2ac032d2-d8cf-47f5-be97-081eb457c5c5">

Reply:
I think you're really close with the top version. the two different type faces clash, which I think is part of what's not working. what I'd try is:
* make the top box a white background, and use poppins as a semibold dark purple in a larger font
* convert the text inside the middle of the box to use poppins regular, or even light
* optional: change the button to dark purple with white text

Reply:
ok middle ground~ ignore the teal magnifying glass. i removed that! 
i'm adding variables for our colors as well into the css !! ok i'm feeling much better about this section now! 
<img width="1215" alt="Screenshot 2024-04-02 at 1 34 46 PM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/72ae9732-1d78-46da-830f-39e54cda490f">